src/graphql/queries/getOfferWithHasApplied.graphql

```diff
@@ -1,5 +1,5 @@
-query ($uuid: ID!) {
-    getOfferVisibleByCurrentApplicant(uuid: $uuid) {
+query getOfferWithHasApplied($uuid: ID!) {
```

Comment:
nombre medio feo, creo que lo llamaria algo como "get offer for applicant" o algo asi